grammar/Python.asdl

```diff
@@ -86,6 +86,7 @@ module LPython
          | ConstantEllipsis(string? kind)
          | ConstantNone(string? kind)
          | ConstantBytes(string value, string? kind)
+         | NONE()
```

Comment:
Why cannot we use ConstantNone?

Reply:
Consider this example:
```py
x = {
    **a,
    None: "Text 3",
}
```
When we use the Constant we get:
```py
└-Module
  |-body=↧
  | └-Assign
  |   |-targets=↧
  |   | └-Name
  |   |   |-id=x
  |   |   └-expr_contextType=Store
  |   |-value=Dict
  |   | |-keys=↧
  |   | | |-ConstantNone
  |   | | | └-kind=()
  |   | | └-ConstantNone
  |   | |   └-kind=()
  |   | └-values=↧
  |   |   |-Name
  |   |   | |-id=a
  |   |   | └-expr_contextType=Load
  |   |   └-ConstantStr
  |   |     |-value="Text 3"
  |   |     └-kind=()
  |   └-type_comment=()
  └-type_ignores=↧
```
is this fine?